storage/src/test/java/tech/pegasys/teku/storage/server/network/DatabaseNetworkTest.java

```diff
@@ -82,4 +86,45 @@ public void shouldNotThrowIfForkAndContractMatch(@TempDir final File tempDir) th
 
     assertDoesNotThrow(() -> DatabaseNetwork.init(networkFile, fork, eth1Address));
   }
+
+  @Test
+  void shouldWriteAndReadDatabaseNetworkWithChainId(@TempDir final File tempDir)
+      throws IOException {
+    final File networkFile = new File(tempDir, "network.yml");
+
+    final Bytes4 fork = dataStructureUtil.randomFork().getCurrentVersion();
+    final Eth1Address eth1Address = dataStructureUtil.randomEth1Address();
+    final Long depositId = dataStructureUtil.randomLong();
+    final DatabaseNetwork databaseNetwork =
+        new DatabaseNetwork(fork.toHexString(), eth1Address.toHexString(), depositId);
+
+    final ObjectMapper objectMapper = new ObjectMapper(new YAMLFactory());
+    objectMapper.writerFor(DatabaseNetwork.class).writeValue(networkFile, databaseNetwork);
+    final DatabaseNetwork readDatabaseNetwork =
+        objectMapper.readerFor(DatabaseNetwork.class).readValue(networkFile);
+
+    assertEquals(fork.toHexString(), readDatabaseNetwork.forkVersion);
+    assertEquals(eth1Address.toHexString(), readDatabaseNetwork.depositContract);
+    assertEquals(depositId, readDatabaseNetwork.depositChainId);
+  }
+
+  @Test
+  void shouldWriteAndReadDatabaseNetworkWithoutChainId(@TempDir final File tempDir)
+      throws IOException {
+    final File networkFile = new File(tempDir, "network.yml");
```

Comment:
just use the string in VersionedDatabaseFactory for filename...

Reply:
`  @VisibleForTesting static final String NETWORK_FILENAME = "network.yml";
`
This is not public. I am unable to access it in the test.